types/cloneable-readable/index.d.ts

```diff
@@ -1,17 +1,22 @@
-// Type definitions for cloneable-readable 1.1
+// Type definitions for cloneable-readable 2.0
 // Project: https://github.com/mcollina/cloneable-readable#readme
-// Definitions by: Nikita Volodin <https://github.com/qlonik>
+// Definitions by: Nikita Volodin <https://github.com/qlonik>, Oleg Vaskevich <https://github.com/vaskevich>
```

Comment:
Can you please place your name on the next line and align with other names?

Reply:
Sure.